main.go

```diff
@@ -43,29 +41,30 @@ BE ON THE LOOKOUT !
 
 `
 	fmt.Println(message)
-	for k:=6; k>0; k--{
-		fmt.Printf("%d ..",k)
+	for k := 6; k > 0; k-- {
+		fmt.Printf("%d ..", k)
 		time.Sleep(time.Second)
 	}
 	fmt.Println("Gooooo!")
 
 	// Creating a random variable to draw one card
 
 	rand.Seed(time.Now().UTC().UnixNano())
-	
+
+	fmt.Printf("first Card %s \n", cards[0])
```

Comment:
You can remove this debugging code after you are happy you have fixed the issue it was helpful for.